### ISSUE TITLE
Update docker development Windows install instructions

### ISSUE THREAD

**Issue by mturoci** [h2oai/wave-apps#115]
Update the Windows install instructions in https://github.com/h2oai/wave-apps/tree/main/docker-development, based on the UNIX ones.

**Comment by InfasMohammed** [comment]
Here are the updated instructions for installing Docker on Windows:

Check the system requirements: Docker Desktop for Windows requires 64-bit Windows 10 Pro, Enterprise, or Education version 1809 or later, with Hyper-V enabled. You also need at least 4GB of RAM.

Download Docker Desktop for Windows: Go to the Docker website and download the Docker Desktop for Windows installer.

Run the installer: Double-click the installer to start the installation process. If prompted, allow the installer to make changes to your system.

Configure Docker: After the installation is complete, Docker Desktop launches automatically. You can configure Docker by clicking the Docker icon in the system tray and selecting "Settings". Here, you can adjust settings such as the amount of memory and CPU Docker is allowed to use.

Verify installation: Open a command prompt or PowerShell window and run the command docker version to verify that Docker is installed and running correctly.

Start using Docker: You can start using Docker by running Docker commands in a command prompt or PowerShell window. You can also use a graphical user interface such as Kitematic to manage your Docker containers.

That's it! With these steps, you should now have Docker up and running on your Windows machine.

**Comment by mturoci** [comment]
Thanks @InfasMohammed! Can you make a PR with these instructions in the Windows section of this [readme](https://github.com/h2oai/wave-apps/tree/main/docker-development)?

**Comment by mturoci** [comment]
Also please verify, if

```sh
git clone https://github.com/h2oai/wave-apps.git
cd wave-apps/docker-development
docker build . -t wave_local_dev
docker run -p 10101:10101 -v $(pwd)/src:/app/src wave_local_dev:latest
```

work on windows as well and successfully allow you to go to `http://localhost:10101` in your browser.

**Comment by devarsh10** [comment]
> Also please verify, if
> 
> ```shell
> git clone https://github.com/h2oai/wave-apps.git
> cd wave-apps/docker-development
> docker build . -t wave_local_dev
> docker run -p 10101:10101 -v $(pwd)/src:/app/src wave_local_dev:latest
> ```
> 
> work on windows as well and successfully allow you to go to `http://localhost:10101` in your browser.

Hey @mturoci, I verified it on Windows and I got the "Hi, Hello World!" message.
Is it expected?

Screenshot:
![image](https://github.com/h2oai/wave-apps/assets/83413047/e0f1fcbb-aa3a-4525-b01c-152276345f33)


Regards,
Devarsh Shah.

**Comment by mturoci** [comment]
Yes @devarsh10, that's right. You can update the README accordingly - the instructions are the same for all OSes.

**Comment by mturoci** [comment]
No. That's covered in https://www.docker.com/. The goal is to make it clear how to run the dockerized app - https://github.com/h2oai/wave-apps/tree/main/docker-development. The wording used may be a bit confusing so we can change `installation` to `set up` maybe.

**Comment by harshjais369** [comment]
> Thanks @InfasMohammed! Can you make a PR with these instructions in the Windows section of this [readme](https://github.com/h2oai/wave-apps/tree/main/docker-development)?

Can you assign me this issue under hacktoberfest?

**Comment by mturoci** [comment]
@harshjais369 you can go ahead. We do not assign issues.

**Comment by denzuko** [comment]
One simpler method is to append docker-development/README.md:20 to include ` / Windows` since docker for desktop on win64 systems does have the same CLI tools that MacOS and Linux has, this also includes WSL.

**Comment by denzuko** [comment]
I have both implemented in https://github.com/h2oai/wave-apps/pull/123
